subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/AbstractTaskPropertyBuilderTest.groovy

```diff
@@ -1,52 +0,0 @@
-/*
- * Copyright 2016 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.gradle.api.internal.tasks
-
-import org.gradle.api.internal.file.FileResolver
-import spock.lang.Specification
-import spock.lang.Unroll
-
-class AbstractTaskPropertyBuilderTest extends Specification {
```

Comment:
Don't we want to keep that test around in some form? For example we could just test `TaskPropertyUtils#checkPropertyName` in a unit test.

Reply:
Good point, I've added it back.